kops

```diff
@@ -208,13 +210,19 @@ def _install_addons(kubecfg_path, args):
             _info('Waiting for cluster to become ready...')
             time.sleep(5)
         else:
+            time.sleep(5)
             break
 
     install_addon('heapster.yaml')
+    time.sleep(5)
```

Comment:
Why do you add sleeps between installing each add-on, I have never seen any need for this?

Reply:
I added it because for some reason kops cli where not ready for the next instruction and makes Python kops script to crash. Probably because of bandwidth where I live?

Reply:
@egimbernat It can't be the `kops` command that fails since `install_addon` calls `kubectl apply`, i.e. it would have to be `kubectl` that would fail. I don't see why `kubectl` would fail to issue requests in rapid succession, could you try asking in the Kubernetes Slack for example to find out why this might happen?

Reply:
we updated the cluster validation functionality which might resolve the issue. https://github.com/coder-society/k8s-aws/pull/10